README.md

```diff
@@ -58,9 +64,10 @@ go install github.com/manuelarte/embeddedstructfieldcheck@latest
 And then use it as:
 
 ```bash
-embeddedstructfieldcheck [--fix]
+embeddedstructfieldcheck [-forbid-mutex=true|false] [--fix]
 ```
 
+- `forbid-mutex`: `true|false` (default `false`) Checks that `sync.Mutex` is not used as an embedded field.
 - `fix`: Fix the case when there is no space between the embedded fields and the regular fields.
 
 ## Resources
```

Comment:
You might add a link in resources

> Exception: Mutexes should not be embedded, even on unexported types. 

https://github.com/uber-go/guide/blob/master/style.md#embedding-in-structs

I feel like forbid-mutex argument is only documenting what is does, but not the why it might be interesting to use it